### PR TITLE
Bugfix: Schema default values mapping

### PR DESCRIPTION
**Pull request by znicholasbrown**
Fixes an issues where the default values coming from mapped schemas were themselves not mapped.

### CODE REVIEW

File: src/services/schemas/utilities.ts [hunk 1]
```diff
@@ -170,7 +170,8 @@ export type ResolverCallback<T> = (schema: T) => T
  * A utility for passing a value through an array of resolver methods.
  */
 export function resolve<T>(value: T, resolvers: ResolverCallback<T>[]): T {
-  return resolvers.reduce((resolved, resolver) => resolver(resolved), value)
+  const resolved = resolvers.reduce((resolved, resolver) => resolver(resolved), value)
+  return { ...resolved, mapped: true }
```
Comment (znicholasbrown) [inline src/services/schemas/utilities.ts:174]: Oops, byproduct of testing, removing this.

File: src/components/FlowRunCreateForm.vue [hunk 2]
```diff
@@ -125,7 +126,7 @@
   }
 
   const combinedParameters = computed(() => {
-    return mapper.map('SchemaValuesResponse', { values: { ...props.deployment.rawParameters, ...props.parameters }, schema: props.deployment.parameterOpenApiSchema }, 'SchemaValues')
+    return mapper.map('SchemaValuesResponse', { values: merge({}, props.deployment.rawParameters, props.parameters), schema: props.deployment.parameterOpenApiSchema }, 'SchemaValues')
```
Comment (znicholasbrown) [inline src/components/FlowRunCreateForm.vue:129]: This is probably a safer way to merge these

Comment (pleek91) [reply src/components/FlowRunCreateForm.vue:129]: +1

File: src/components/SchemaInput.vue [hunk 3]
```diff
@@ -78,8 +78,9 @@
   })
 
   const schemaDefaultValues = getSchemaDefaultValues(props.schema)
-  const defaultValues = merge({}, schemaDefaultValues, props.modelValue ?? {})
-  const unmappedDefaultValues = mapper.map('SchemaValues', { values: defaultValues, schema: props.schema }, 'SchemaValuesRequest')
+  const unmappedModelValue = mapper.map('SchemaValues', { values: props.modelValue ?? {}, schema: props.schema }, 'SchemaValuesRequest')
+  const unmappedDefaultValues = merge({}, schemaDefaultValues, unmappedModelValue)
```
Comment (pleek91) [inline src/components/SchemaInput.vue:82]: I believe `getSchemaDefaultValues` returns `SchemaValues` but you're merging them with `SchemaValuesRequest`. I think you probably want to merge them with the already mapped modalValue

**Review by pleek91 — APPROVED** [review]

**Pull request merged**